orderer/common/types/channelinfo.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package types
+
+// ErrorResponse carries the error response an HTTP request.
+// This is marshaled into the body of the HTTP response.
+type ErrorResponse struct {
+	Error string `json:"error"`
+}
+
+// ChannelList carries the response to an HTTP request to List all the channels.
+// This is marshaled into the body of the HTTP response.
+type ChannelList struct {
+	Size          int                `json:"size"`          // The number of channels
+	SystemChannel string             `json:"systemChannel"` // The system channel name, empty if doesn't exist
+	Channels      []ChannelInfoShort `json:"channels"`      // All channels
+}
+
+// ChannelInfoShort carries a short info of a single channel.
+type ChannelInfoShort struct {
+	Name string `json:"name"` // The channel name
+	URL  string `json:"url"`  // The channel relative URL (no Host:Port)
+}
+
+// ChannelInfo carries the response to an HTTP request to List a single channel.
+// This is marshaled into the body of the HTTP response.
+type ChannelInfo struct {
+	Name            string `json:"name"`            // The channel name
+	URL             string `json:"url"`             // The channel relative URL (no Host:Port)
+	ClusterRelation string `json:"clusterRelation"` // Whether it is a “member” or ”follower”, case insensitive
+	Status          string `json:"status"`          // ”onboarding” or ”active”, case insensitive
+	Height          uint64 `json:"height"`          // Current block height
+	BlockHash       []byte `json:"blockHash"`       // Last block hash
```

Comment:
What are clients expected to do with the block hash?

Reply:
For example match it with other OSNs response?

Reply:
What do you expect to find, a fork in the Blockchain? 

And besides, the intended users of this interface is local administrator, you only have access to your own nodes anyway, so if a malicious leader is going to fork you, it is going to fork all your organizations nodes now that it knows you can compare hashes.

Reply:
By the way let's encode this in hexa or base64 otherwise the terminal might puke funny characters if it's a byte slice.

Reply:
It is marshaled into a JSON object as base64 by default. 
But I do have to implement a Stringer to make it log friendly, that is correct.

Reply:
OK but I'm still struggling to understand why you need the hash. It only makes the output more verbose, and I doubt you can do anything useful with this hash. 

Generally, I am reluctant adding information or functionality that I am not sure how they are going to be used.

Reply:
Adding the block-hash was a review comment I got on the [design document](https://docs.google.com/document/d/1kYoicS_Z59gNvmuuB7VFZ6WsOL31C34dWMeOFQiIygA/edit?usp=sharing), so I added it. I don't mind taking it off.